library/src/lib/popover/popover-directive/popover-container.scss

```diff
@@ -166,3 +166,6 @@
         }
     }
 }
+::ng-deep .fd-dropdown {
+    position: static;
```

Comment:
I was more thinking of putting it inside popover.component.scss (if that doesn't exist, we probs should have one). That way its on the same level as where we actually use the class, and it would also **not** be needed to put ::ng-deep I think